vault/changelog.release.2022-02-08.md

```diff
@@ -0,0 +1,149 @@
+---
+id: E1Zgn2za7n3QVfm3MtIBe
+title: '0.81'
+desc: ''
+updated: 1644341926802
+created: 1644289547625
+---
+
+Dendron 0.81 has sprouted  🌱
+
+[[Pod V2|dendron://dendron.dendron-site/dendron.topic.pod-v2]], the new and improved Dendron pod system, is now in preview! Improvements include flexibility that helps you export a single note, a hierarchy, a vault, or your entire workspace. Pod V2 also includes customizable export mappings where you can select frontmatter fields that you want to export.
```

Comment:
can we leave out customizable export mappings? that is only supported on airtable today (@Harshita-mindfire is working on making this generic)